src/sentry/eventstream/base.py

```diff
@@ -97,6 +108,7 @@ def insert(
         received_timestamp: float,
         skip_consume: bool = False,
         group_states: Optional[GroupStates] = None,
+        **kwargs: Any,
```

Comment:
I don't think you should do this. Why can't you use: `Event.get_event_type() == "transaction"` as all the other checks in the post process forwarder do?